sympy/utilities/lambdify.py

```diff
@@ -3,7 +3,7 @@
 lambda functions which can be used to calculate numerical values very fast.
 """
 
-from typing import Any, Dict as tDict, Iterable, Union as tUnion, TYPE_CHECKING
```

Comment:
You removed Dict and Any from the import but they don't seem to be removed in the file. Were they unused or is this removing too much?

Reply:
My linter said they were unused. I'll have to check, maybe they weren't.

Reply:
There are a ton of comments like this:

```python
# type: tDict[str, str]
```

I'm not sure if they actually have functionality.

Reply:
I removed all those comments.